Mechanics/Skill.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Text;
+
+namespace SpaceOpera.Mechanics
+{
+    class Skill
+    {
+        private SkillName name;
+
+        public SkillName GetName()
```

Comment:
Best would be to move getters and setters to the end of a class since they are auto-generated and thus neither interesting nor important